sql/sql_expression.h

```diff
@@ -1301,14 +1301,24 @@ PREDECLARE_VALUE_DESCRIPTION(TupleExpression);
 */
 struct GenerateRowsWhereFunction {
 
+    enum Complexity {        
+       
+        CONSTANT = 0,
+        BETTER_THAN_TABLESCAN,        
+        UNFILTERED_TABLESCAN,
```

Comment:
What would be an example of `UNFILTERED_TABLESCAN`? I could not find any example in the test case.